submissions/Vlada-khom/js-dom/main.js

```diff
@@ -0,0 +1,130 @@
+        back from the face toward the center of the back of the skull. The breed originated in Lakewood, 
+        California, as the result of a spontaneous mutation.`
+    ],
+    image: "./db/curl.jpg",
+  },
+  {
+    name: "British Longhair",
+    id: '4890c3f5-3691-4e08-afca-783a2343c187',
+    description: [
+      `The British Longhair is a medium-sized, semi-long-haired breed of domestic cat, 
+        originating in Great Britain.`,
+      `The British Longhair is a longer-haired development from the 
+        longstanding British Shorthair breed. In the mid-20th century, British Shorthairs were interbred 
+        with imported long-haired varieties, like the Turkish Angora and what today is called the 
+        Traditional Persian, with an aim to producing more stout and round-faced stock, while retaining 
+        the short coat. As a result of this hybridization, British catteries have frequently produced 
+        (generally unwanted) semi-long-haired offspring among their litters. In more recent years, 
+        these have been intentionally bred (often outside the UK) to each other and sometimes to 
+        standard British Shorthairs, to establish a consistent, formalized British Longhair breed.`,
+    ],
+    image: "./db/british-long.jpg",
+  },
+  {
+    name: "British Shorthair",
+    id: '2263aab2-f0b8-4ce9-be9d-94c1fc3107e2',
+    description: [
+      `The British Shorthair is the pedigreed version of the traditional British domestic 
+        cat, with a distinctively stocky body, dense coat, and broad face. The most familiar colour 
+        variant is the "British Blue", with a solid grey-blue coat, orange eyes, and a medium-sized 
+        tail. The breed has also been developed in a wide range of other colours and patterns, including 
+        tabby and colourpoint.`,
+      `It is one of the most ancient cat breeds known. In modern times, it remains the most popular 
+        pedigreed breed in its native country, as registered by the UK's Governing Council of the Cat 
+        Fancy (GCCF). A quarter of all kittens registered with the GCCF each year are British 
+        Shorthairs, making the British the most popular pedigree cat in the UK.`,
+      `The breed's good-natured appearance and relatively calm temperament make it a frequent media 
+        star, notably as the inspiration for John Tenniel's famous illustration of the Cheshire Cat 
+        from Alice in Wonderland. The Cat Fanciers' Association profile reads: "When gracelessness is 
+        observed, the British Shorthair is duly embarrassed, quickly recovering with a 'Cheshire cat 
+        smile'.`,
+    ],
+    image: "./db/british-short.jpg",
+  },
+];
+
+
+
+function main() {
+  const sideMenu = document.querySelector(".sideMenu");
+  const menuList = document.createElement("ul");
+  menuList.className = "menuList";
+  data.forEach((el) => {
+    menuList.innerHTML += `<li><div class="itemMenu" data-id="${el.id}">${el.name}</div></li>`;
```

Comment:
Set more informative names.  `catList.map(cat => {})`
I recommend to use `Array.map()` or `Array.reduce`. You use one of these methods in the code below.